main.c

```diff
@@ -159,7 +159,8 @@ int NMEA_message_handler(int sock)
 			// Compute Vario
 			vario = ComputeVario(vkf.x_abs_, vkf.x_vel_);
 			
-			if (config.output_POV_P_Q == 1)
+			if (config.output_POV_P_Q == 1
+				|| io_mode.sensordata_from_file == TRUE)
```

Comment:
Why do you overwrite the NMEA config if data airdata is coming from file ??

Reply:
I need the part under the if not only when sensors are present,
but also when data comes from file.

Reply:
> Please split up the commits in logic functions.

Done.